Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apk add --update --no-cache bash make curl coreutils libc6-compat tar xz
 
 COPY --from=cfssl /go/bin/ ${INSTALL_PATH}/
 
-ADD . /packages
-RUN make -C /packages/install/ all
+COPY . /packages
+RUN mkdir -p /packages/tmp
+RUN make -C /packages/install/ terraform
```

Comment:
Is this to test that it’s working?

Reply:
Yes, should be "all" instead of terraform. Will fix.

Reply:
Copy instead of Add is called out as best practice in several places, including Docker [docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy).
Explicitly making the tmp directory in case it gets lost by someone's `.dockerignore` again which is what was happening to us.